propka/ligand_pka_values.py

```diff
@@ -159,25 +163,32 @@ def get_marvin_pkas_for_molecule(self, atoms, filename='__tmp_ligand.mol2',
             [self.cxcalc, filename]+options.split(), stdout=subprocess.PIPE,
             stderr=subprocess.PIPE).communicate()
         if len(errors) > 0:
-            info('***********************************************************'
-                 '*********************************************')
-            info('* Warning: Marvin execution failed:                        '
-                 '                                            *')
-            info('* {0:<100s} *'.format(errors))
-            info('*                                                          '
-                 '                                            *')
-            info('* Please edit the ligand mol2 file and re-run PropKa with '
-                 'the -l option: {0:>29s} *'.format(filename))
-            info('***********************************************************'
-                 '*********************************************')
+            _LOGGER.info(
+                '***********************************************************'
+                '*********************************************')
+            _LOGGER.info(
+                '* Warning: Marvin execution failed:                        '
+                '                                            *')
+            _LOGGER.info('* {0:<100s} *'.format(errors))
+            _LOGGER.info(
+                '*                                                          '
+                '                                            *')
+            _LOGGER.info(
```

Comment:
I would make this `logger.error` or at least `logger.warn`.

Also, I would replace the nasty `sys.exit()` with a `raise OSError` (or something better... ) and have the exception mirror the message. A problem with the logging system is that people might not actually see the message unless they enable logging.

Reply:
Addressed in https://github.com/jensengroup/propka/pull/109/commits/c2816230780e81e75ae9093388c746cf345f5abf and https://github.com/jensengroup/propka/pull/109/commits/de020599d31d782a9dda40afeb662148df7448bd